readthedocs/settings/docker_compose.py

```diff
@@ -203,6 +203,11 @@ def DATABASES(self):  # noqa
     AWS_S3_ENDPOINT_URL = "http://storage:9000/"
     AWS_QUERYSTRING_AUTH = False
 
+    STRIPE_SECRET = os.environ.get("RTD_STRIPE_SECRET")
+    STRIPE_PUBLISHABLE = os.environ.get("RTD_STRIPE_PUBLISHABLE")
+    STRIPE_TEST_SECRET_KEY = STRIPE_SECRET
+    DJSTRIPE_WEBHOOK_SECRET = os.environ.get("RTD_DJSTRIPE_WEBHOOK_SECRET")
```

Comment:
We will need some default values for this. Otherwise, it fails when running the application:

```
SystemCheckError: System check identified some issues:

CRITICALS:
?: (djstripe.C001) Could not find a Stripe API key.
	HINT: Add STRIPE_TEST_SECRET_KEY and STRIPE_LIVE_SECRET_KEY to your settings.
```

@agjohnson can you add the default values we used to have?

Reply:
Well, one of the reasons I added this is that core team should all be using these secrets in their local environment. We should all be testing/developing with the same setup.

But I will add a default for anyone that is not core team.